src/lib/currency/signed_poly.ml

```diff
@@ -0,0 +1,15 @@
+open Core_kernel
+
+module Signed = struct
```

Comment:
Nit: I think this module is unnecessary, `Signed_poly.t` seems reasonable enough.